articles/060_ros_middleware_interface.md

```diff
@@ -330,6 +314,8 @@ The described concept has been implemented in the following packages:
   - the functions are declared in [rms/rmw.h](https://github.com/ros2/rmw/blob/master/rmw/include/rmw/rmw.h)
   - the handles are defined in [rmw/types.h](https://github.com/ros2/rmw/blob/master/rmw/include/rmw/types.h)
 
+- the package [rmw_fastrtps_cpp](https://github.com/ros2/rmw_fastrtps/tree/master/rmw_fastrtps_cpp) implements the middleware interface using [eProsima Fast-RTPS](http://www.eprosima.com/index.php/products-all/eprosima-fast-rtps) based on the introspection type support
+
```

Comment:
> rosidl_generator_dds_idl is used by all rmw implementations based on static code (e.g. opensplice, coredx) and not only Connext

Fixed in 15b76d52b004f2d4e1f804dee743b605c639ce10 by moving it above the vendor specific bullets.

> not sure if we should flag non supported implementations one way or another because we could expect that rmw_connext_dynamic and rmw_opensplice are at feature parity and receive the same amount of effort as the other ones

Each repository should already mention that. I don't see a need to duplicate that information here (and require it to be kept up-to-date in the future).